dist.ini

```diff
@@ -1,8 +1,12 @@
 name    = Log-Any-Adapter-DERIV
-author  = Deriv Services Ltd <DERIV@cpan.org>
+author  = binary.com <BINARY@cpan.org>
 license = Perl_5
-copyright_holder = Deriv Services Ltd
-copyright_year   = 2020
-main_module	= lib/Log/Any/Adapter/DERIV.pm
+copyright_holder = Binary Services Ltd
```

Comment:
Are we sure this should be binary and not Deriv?

Reply:
I'm sure, because we have already released it to PAUSE with binary name:  https://metacpan.org/pod/Log::Any::Adapter::DERIV

Reply:
okay!